ember-scoped-css/build.mjs

```diff
@@ -55,7 +55,7 @@ await esbuild.build({
 
 // Runtime
 await esbuild.build({
-  entryPoints: ['src/runtime/test-support.ts'],
+  entryPoints: ['src/runtime/test-support.ts', 'src/runtime/index.ts'],
```

Comment:
part 1 of the fix is here, the runtime module is required to be resolved, even if never used